src/routes/docs/products/functions/execute/+page.markdoc

```diff
@@ -485,6 +485,285 @@ Here are some cron expressions for common intervals:
 | `0 0 * * *`      | Every day at 00:00    |
 | `0 0 * * 1`      | Every Monday at 00:00 |
 
+# Delayed executions {% #delayed-executions %}
+
+You can also delay function executions, which trigger the function only once at a future date and time. You can schedule a function execution using the Appwrite Console or a Server SDK.
+
+{% tabs %}
+{% tabsitem #console title="Console" %}
+To schedule an execution, navigate to **Your function** > **Executions** > **Execute now** > **Schedule** in the Appwrite Console.
+
+{% only_dark %}
+![Add a SMTP provider](/images/docs/functions/execution/dark/scheduled-execution-function.png)
+{% /only_dark %}
+{% only_light %}
+![Add a SMTP provider](/images/docs/functions/execution/scheduled-execution-function.png)
+{% /only_light %}
+{% /tabsitem %}
+{% tabsitem #sdk title="Server SDK" %}
+You can also schedule your function executions using a supported [runtime](/docs/products/functions/runtimes#available-runtimes).
```

Comment:
Again, you don't schedule an execution using a runtime. You schedule an execution using an SDK or the API. It's then executed inside a runtime.